spec/xcpretty/formatters/simple_spec.rb

```diff
@@ -20,6 +20,11 @@ module XCPretty
         "> Building Pods/The Spacer [Debug]"
       end
 
+      it "formats analyze target/project/configuration with target" do
+        @formatter.format_analyze_target("The Spacer", "Pods", "Debug").should ==
+        "> Building and Analyzing Pods/The Spacer [Debug]"
```

Comment:
Use 2 (not 0) spaces for indenting an expression spanning multiple lines.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.